### DISCUSSION TITLE
How to accept my submission of GitHub Education

### DISCUSSION THREAD

**Discussion by Bill-Flash** [community/community#50898] [GitHub Education]
Since my school is a joint school, I use my school ID from another country. But now my current location is not in this country, and I use a digital transcript of my certificate of attendance, verifying my academic affiliation kept failing.  

Besides, I think my school-provided academic affiliation document indicates where I study, and they are the same as my current location.

**Comment by samus-aran** [comment]
Hi @Bill-Flash 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of your latest rejection reasons? This might help someone give you a nudge in the right direction.
